packages/components/src/Components/DateFormat/DateFormat.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { dateStringByType } from './helper';
+
+export default function DateFormat({ date, type = 'relative' }) {
+    const dateObj = date instanceof Date === 'Date' ? date : new Date(date);
```

Comment:
This should be `date instanceof Date` only, the comparision to `'Data'` is redundant and will actually break the condition.